src/tools/auth0/handlers/emailProvider.ts

```diff
@@ -34,9 +36,10 @@ export default class EmailProviderHandler extends DefaultHandler {
 
     let existing = await this.getType();
 
+    // TODO: HTTP DELETE on emails/provider is not public, so not part of our vNext SDK.
     if (Object.keys(emailProvider).length === 0) {
       if (this.config('AUTH0_ALLOW_DELETE') === true) {
-        await this.client.emailProvider.delete();
+        // await this.client.emails.delete();
```

Comment:
This could technically be a breaking change but it also seems like this never worked to begin with. We may be able to replicate a delete-like functionality to the best of our ability.